pkgs/development/python-modules/dropbox/default.nix

```diff
@@ -39,6 +39,8 @@ buildPythonPackage rec {
   postPatch = ''
     substituteInPlace setup.py \
       --replace "'pytest-runner == 5.2.0'," ""
+    substituteInPlace dropbox/dropbox_client.py \
+      --replace "__version__ = '0.0.0'" "__version__ = '${version}'"
```

Comment:
Should we report this error upstream?

Also, it is safe to assume this was a mistake right? Probably add a comment here to remove it on next release (or at least check if necessary).

Reply:
Yeah I just did that at https://github.com/dropbox/dropbox-sdk-python/issues/425 .
Depending on the answer I get I will add a comment to remove or keep (this substitution should be safe to have around as a fallback).

Reply:
I would still leave a comment here at least pointing to the issue, just for context in the future.

Reply:
Package was fixed upstream but the same tag/version was re-used so I updated the hash instead to get the latest. This fixes `maestral` and friends (again).